flytekit/clients/friendly.py

```diff
@@ -5,16 +5,41 @@
 from flyteidl.admin import task_pb2 as _task_pb2, common_pb2 as _common_pb2, workflow_pb2 as _workflow_pb2, \
     launch_plan_pb2 as _launch_plan_pb2, execution_pb2 as _execution_pb2, node_execution_pb2 as _node_execution_pb2, \
     task_execution_pb2 as _task_execution_pb2, project_pb2 as _project_pb2
+from flyteidl.core import identifier_pb2 as _identifier_pb2
 
 from flytekit.clients.raw import RawSynchronousFlyteClient as _RawSynchronousFlyteClient
 from flytekit.models import filters as _filters, common as _common, launch_plan as _launch_plan, task as _task, \
     execution as _execution, node_execution as _node_execution
 from flytekit.models.core import identifier as _identifier
 from flytekit.models.admin import workflow as _workflow, task_execution as _task_execution
+from flytekit.common.exceptions.user import FlyteAssertion as _FlyteAssertion
 
 
 class SynchronousFlyteClient(_RawSynchronousFlyteClient):
 
+    def register_entities(self, id_entity_list):
```

Comment:
I think this is a little weird to have in the client because it is a non-transactional method and doesn't map directly to a single API call.

Reply:
i can definitely move this back to the core cli code.